backend/src/main/resources/db/migration/V26__Add_user_info_columns_table.sql

```diff
@@ -0,0 +1,8 @@
+ALTER TABLE users ADD COLUMN wins INT NOT NULL DEFAULT 0;
+ALTER TABLE users ALTER wins DROP DEFAULT;
+
+ALTER TABLE users ADD COLUMN losses INT NOT NULL DEFAULT 0;
+ALTER TABLE users ALTER losses DROP DEFAULT;
+
+ALTER TABLE users ADD COLUMN tournaments INT NOT NULL DEFAULT 0;
+ALTER TABLE users ALTER tournaments DROP DEFAULT;
```

Comment:
Why not keep the defaults here since they make sense when creating a new user

Reply:
By convention "default" values are mandated by the server, not the database.